quickproject.lisp

```diff
@@ -39,10 +38,12 @@ necessary. *DEFAULT-PATHNAME-DEFAULTS* bound to the newly created
 project directory.")
 
 (defun matches-template-p (pathname template)
+  "Compare file name and type (ie. `name.type') parts."
```

Comment:
Should be "Returns true if PATHNAME matches the pathname TEMPLATE in both name and type."